src/components/ListItem.jsx

```diff
@@ -1,5 +1,62 @@
 import './ListItem.css';
+import { useEffect, useState } from 'react';
+import { updateItem } from '../api/firebase';
+import { ONE_DAY_IN_MILLISECONDS } from '../utils/dates';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ name, listPath, itemId, dateLastPurchased }) {
+	const [isChecked, setIsChecked] = useState(false);
+	const [expired, setExpired] = useState();
+
+	const handleChange = async () => {
```

Comment:
Nice implementation here! I know this isn't explicitly in the AC, but I think this provides a better user experience to make sure a user doesn't accidentally check something.